frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-workspaces.feature

```diff
@@ -30,9 +30,10 @@ Feature: Workspaces
             Given user created pipeline with workspace
              When user selects "Start" option from kebab menu for pipeline "test-workspace-pipeline"
               And user fills the Parameters in Start Pipeline modal
-              And user selects volume type "<volume_tyoe>" from workspaces dorp down
+              And user selects volume type "<volume_tyoe>" from workspaces drop down
               And user selects Start button
              Then user will be redirected to Pipeline Run Details page
+              And user will see volume_tyoe mentioned in the Workspace Resources section of Pipeline Run Details page
```

Comment:
```suggestion
              And user will see volume_type mentioned in the Workspace Resources section of Pipeline Run Details page
```
You guys need spellcheck to make sure you aren't later having to fix typos (like you did for "dorp" down above). @makambalaji not sure what we can do here from an automated tooling point of view. 

@sanketpathak there is also the same typo on line 29 & 33.

Reply:
Also shouldn't this be "<volume_type>"?

Reply:
`<volume_tyoe>` can work here as this is used as a variable name to give example values mentioned below the `Scenario Outline`

Reply:
I'm a bit confused... are you suggesting you don't make the typo correction? @sanketpathak

Reply:
> I'm a bit confused... are you suggesting you don't make the typo correction? @sanketpathak

@sanketpathak ?